crates/evm/src/env.rs

```diff
@@ -9,6 +9,18 @@ pub struct EvmEnv {
     pub block_env: BlockEnv,
 }
 
+impl Default for EvmEnv {
+    fn default() -> Self {
+        Self {
+            cfg_env_with_handler_cfg: CfgEnvWithHandlerCfg {
+                cfg_env: Default::default(),
+                handler_cfg: Default::default(),
```

Comment:
can we add a sanity doc here that handler_cfg::default will configure optimism if revm's optimism feature is enabled?